assets/css/lesson5.css

```
@@ -0,0 +1,762 @@
+*{
+    margin: 0px;
+    padding: 0px;
+    color: black;
+    text-decoration: none;
+    font-size: 14px;
+    font-family: Cambria;
+    list-style-type: none;
+}
+
+.div1{
```

```suggestion
.div1 {
```
để thế này có vẻ đẹp hơn này